patches/net/minecraft/server/level/ServerPlayerGameMode.java.patch

```diff
@@ -82,11 +77,18 @@
          }
 +    }
 +
-+    private boolean removeBlock(BlockPos p_180235_1_, boolean canHarvest) {
-+        BlockState state = this.level.getBlockState(p_180235_1_);
-+        boolean removed = state.onDestroyedByPlayer(this.level, p_180235_1_, this.player, canHarvest, this.level.getFluidState(p_180235_1_));
++    /**
++     * Patched-in method that handles actual removal of blocks for {@link #destroyBlock(BlockPos)}.
++     *
++     * @param pos The block pos of the destroyed block
++     * @param state The state of the destroyed block
++     * @param canHarvest If the player breaking the block can harvest the drops of the block
++     * @return If the block was removed, as reported by {@link BlockState#onDestroyedByPlayer}.
++     */
++    private boolean removeBlock(BlockPos pos, BlockState state, boolean canHarvest) {
```

Comment:
Should this method have an EXC entry for mapping `pos` back to `p_180235_1_`?

Reply:
No, the original method was a patch as well.  Not sure why the parameter name was obfuscated in that case.